Epsilon.Host.Frontend/src/components/Competence/KpiTable.vue

```diff
@@ -22,7 +22,7 @@
                 <div
                     v-for="submission of store.state.filterdSubmissions.filter(
                         (s) =>
-                            s.results.filter((r) => r.outcome.id === outcome)
+                            s.results?.filter((r) => r.outcome.id == outcome)
```

Comment:
Again, maybe just check if results is null before hand, so you know that in a specific scope it is definetly not null. That way you can more easily recognize errors as well.

Reply:
I changed it. But this becomes a mess if we want to do it to ever tying. I think it would be best to ensure the response from the backend. Better an empty array then a mess with question marks or if statements